### DISCUSSION TITLE
CANCELLED" Next GE meeting: Monday Jan 23, *18:30* UTC (**note the new time**)

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#99] [Team Posts]
Since nobody objected to the change of time and it is better for our host @wsun106 , let's move the grammar engineering meetings to **18:30 UTC** for now. 

The next one is this coming Monday, January 23 (which is during the Wordnet conference, and I think other people might also have a conflict, so this may end up being a very small meeting).

Zoom: 926 4912 4250
Password: 674288

https://cl-cam-ac-uk.zoom.us/j/92649124250?pwd=L2YzUE1pU3NHSmlsVDVnS203VmpNdz09

**Comment by fcbond** [comment]
Hi,

I will definitely not make next Monday's meeting.

On Wed, 18 Jan 2023 at 13:35, Olga Zamaraeva ***@***.***>
wrote:

> Since nobody objected to the change of time and it is better for our host
> @wsun106 <https://github.com/wsun106> , let's move the grammar
> engineering meetings to *18:30 UTC* for now.
>
> The next one is this coming Monday, January 23 (which is during the
> Wordnet conference, and I think other people might also have a conflict, so
> this may end up being a very small meeting).
>
> Zoom: 926 4912 4250
> Password: 674288
>
>
> https://cl-cam-ac-uk.zoom.us/j/92649124250?pwd=L2YzUE1pU3NHSmlsVDVnS203VmpNdz09
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/42>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRTH4VBWDOVHMFBERPDWS7PPTANCNFSM6AAAAAAT7A4UKQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by olzama** [comment]
@danflick I think you and I might be the only ones available for this one (from the usual attendees). I do have things to talk to you about (including the huggingface dataset which I mentioned in my email to you) but let me know whether you can/want to make it this coming week. If not, we will just cancel this one.

**Comment by arademaker** [comment]
I will not be able to join next Monday. Traveling to GWC conference. Can we make in two weeks from now.

**Comment by arademaker** [comment]
huggingface dataset? Please postpone this discussion so we can all participate!

**Comment by danflick** [comment]
Sorry, I will also be traveling on Monday the 23rd, so let's wait until the
next one, and include the HuggingFace topic then.

 Dan

**Comment by olzama** [comment]
Let's cancel this one then! @wsun106, since almost nobody can make it, let's not meet on the 23rd!
